mem3_helper.py

```diff
@@ -77,7 +82,7 @@ def connect_the_dots(names):
 
 def check_membership_forever():
     creds = (os.getenv("COUCHDB_USER"), os.getenv("COUCHDB_PASSWORD"))
-    expected_peers_count = os.getenv("COUCHDB_CLUSTER_SIZE")
+    expected_peers_count = int(os.getenv("COUCHDB_CLUSTER_SIZE"))
```

Comment:
Int conversion will fail in case `COUCHDB_CLUSTER_SIZE` is not set.

Reply:
Solved